lib/helper.js

```diff
@@ -94,6 +94,32 @@ function stubComponent(tag, children, showDataProps) {
 }
 
 /**
+ * Wraps component in a context creating component
+ *
+ * @method withContext
+ * @param {ReactComponent} Component class
+ * @return {ReactElement} react class with context setup
+ **/
+function withContext(context, Component) {
+    var Elem = React.createFactory(Component);
+    var childContextTypes = {};
+
+    for (var key in context) {
+        childContextTypes[key] = React.PropTypes.any;
+    }
+
+    return React.createClass({
+        childContextTypes: childContextTypes,
+        getChildContext: function () {
+            return context;
+        },
+        render: function () {
+            return Elem(this.props);
```

Comment:
I tried this, but using `createElement` prevented `context` from being properly inherited later